### PR TITLE
Added friendly quarg hails and changed a few other hails

### PR DESCRIPTION
**Pull request by Gkramme**

### CODE REVIEW

File: data/hails.txt [hunk 1]
```diff
@@ -735,7 +736,7 @@ phrase "friendly navy"
 		"For honor! For glory!"
 		"Answer the call of duty! Join the Navy!"
 		"If you have courage, if you have honor, join the Navy!"
-		"We need YOU for the Navy!"
+               "Uncle Sam needs YOU for the Navy!
```
Comment (jafdy) [inline data/hails.txt]: Erm did i miss a reference here.
Keep the old hail, It's one of the nice ones.

File: data/hails.txt [hunk 2]
```diff
@@ -996,7 +998,40 @@ phrase "friendly pug"
...
-
+phrase "friendly quarg"
+        word 
+                "We "
+                "The Quarg "
+		"Our mission is to "
+		"The Drak have asked us to "
+		"Our species will "
+		"Our job is to "
+		"We wish to "
```
Comment (toilethinges) [inline data/hails.txt]: A bit of spacing issues before line 1003 and 1004.

File: data/hails.txt [hunk 3]
```diff
@@ -996,7 +998,40 @@ phrase "friendly pug"
...
+		"bring about "
+	word
+		"peace"
+		"healing"
+		"tranqulity"
+		"freedom"
+		"an end to hate"
+		"an end to violence"
+		"an end to warfare"
+                "peace in this part of our galaxy"
```
Comment (Amazinite) [inline data/hails.txt]: Here too.

**Comment by mdsmestad** [comment]
It's doesn't really make sense to say "Uncle Sam needs YOU for the Navy!" because the navy of the future isn't American.

**Comment by Gkramme** [comment]
It's supposed to be an easter egg. It also hints at the Republic's past...

**Comment by Amazinite** [comment]
It feels like you're building your own lore here. America ruled the world in the past?

**Comment by Gkramme** [comment]
It would be nice to have a storyline that talks about the events that long predate the character's birth...

**Comment by mdsmestad** [comment]
@Gkramme Well I see where you were going now..but it isn't going to be obvious to most players

**Comment by Amazinite** [comment]
I think most of the past is suppose to be murky. Especially given what it says at the top of the historical timeline.

**Comment by Gkramme** [comment]
Well, it would be nice to know what happened so long ago for those players who really want some backstory...

**Comment by Gkramme** [comment]
@endless-sky any issues you see?

**Comment by endless-sky** [comment]
I'm not sure I want to imply that the US is the dominant government on Earth. 

The Navy hail changes don't really fix the awkwardness of some of the combinations, or the original problem of "the world" not making sense for an interplanetary navy.

**Comment by Gkramme** [comment]
Well, maybe the Republic was origionally founded off the rules of the USA, so they would maybe use their old traditions?

**Comment by Amazinite** [comment]
If anything, they would likely be founded off of something like NATO that is an alliance of countries, not a single country itself.

File: data/hails.txt (hunk 2 above)
Comment (Gkramme) [inline data/hails.txt]: Can you close this @toilethinges?

**Comment by Amazinite** [comment]
You need to go into the government.txt and set the Quarg to actually use the hails.

**Comment by toilethinges** [comment]
You should probably delete the Navy hails, as Amazinite already had a pull merged fixing them.

**Comment by Amazinite** [comment]
And that's likely what is causing the conflict. Try merging the master into your branch.

**Comment by Gkramme** [comment]
I don't think I can do that on a iPad

**Comment by Gkramme** [comment]
@endless-sky what problems do you see?

**Comment by jafdy** [comment]
sorting out the conflicts would help.

**Comment by Gkramme** [comment]
What conflicts do you see?

**Comment by jafdy** [comment]
All the Navy hails.

**Pull request closed without merging**